src/main/java/server/Server.java

```diff
@@ -2,27 +2,32 @@
 
 import com.beust.jcommander.JCommander;
 import com.beust.jcommander.ParameterException;
+import main.java.application.App;
+import main.java.application.Routes;
 import main.java.server.CLI.Args;
 
 import java.io.IOException;
 import java.net.ServerSocket;
 import java.net.Socket;
 
 class Server {
-    private final Integer port;
+    private App app;
+    private Integer port;
     private boolean serverIsRunning = true;
 
-    private Server(Integer port) {
+    public Server(Integer port, App app) {
         this.port = port;
+        this.app = app;
     }
 
     public static void main(String[] args) {
             Args clArgs = new Args();
             JCommander jcParser = new JCommander(clArgs);
+            App app = new App(Routes.getROUTES());
```

Comment:
That has me wondering, though - why does the Server's `main` need to initialize an `App` (and specify what routes the app handles)?